### PR TITLE
Feat: API 7.9

### PR DESCRIPTION
**Pull request by Poolitzer**

### CODE REVIEW

File: telegram/_bot.py [hunk 1]
```diff
@@ -9274,8 +9282,120 @@ async def send_paid_media(
...
+            api_kwargs=api_kwargs,
         )
 
+        return ChatInviteLink.de_json(result, self)  # type: ignore[return-value]
+
+    async def edit_chat_subscription_invite_link(
+        self,
+        chat_id: Union[str, int],
+        invite_link: Union[str, "ChatInviteLink"],
+        name: Optional[str] = None,
```
Comment (harshil21) [inline telegram/_bot.py:9353]: I don't get this method. What part are we editing with the invite link?

File: telegram/_payment/stars.py [hunk 2]
```diff
@@ -310,20 +311,30 @@ class TransactionPartnerUser(TransactionPartner):
...
+            media bought by the user.
 
     Attributes:
         type (:obj:`str`): The type of the transaction partner,
             always :tg-const:`telegram.TransactionPartner.USER`.
         user (:class:`telegram.User`): Information about the user.
         invoice_payload (:obj:`str`): Optional. Bot-specified invoice payload.
+        paid_media (Sequence[:class:`telegram.PaidMedia`]): Optional. Information about the paid
+            media bought by the user.
+
```
Comment (harshil21) [inline telegram/_payment/stars.py:324]:
```suggestion
            media bought by the user.
            
            .. versionadded:: NEXT.VERSION

    Attributes:
        type (:obj:`str`): The type of the transaction partner,
            always :tg-const:`telegram.TransactionPartner.USER`.
        user (:class:`telegram.User`): Information about the user.
        invoice_payload (:obj:`str`): Optional. Bot-specified invoice payload.
        paid_media (Sequence[:class:`telegram.PaidMedia`]): Optional. Information about the paid
            media bought by the user.

            .. versionadded:: NEXT.VERSION
```

File: telegram/constants.py [hunk 3]
```diff
@@ -3096,3 +3102,22 @@ class BackgroundFillType(StringEnum):
...
+    """
+
+    __slots__ = ()
+
+    SUBSCRIPTION_PERIOD = 2592000
+    """:obj:`int`: The number of seconds the subscription will be active."""
+    MIN_PRICE = 1
+    """:obj:`int`: Amount of stars a user pays, minimum amount the subscription can be set to."""
+    MAX_PRICE = 2500
+    """:obj:`int`: Amount of stars a user pays, maxcimum amount the subscription can be set to."""
```
Comment (harshil21) [inline telegram/constants.py:3123]:
```suggestion
    """:obj:`int`: Amount of stars a user pays, maximum amount the subscription can be set to."""
```

File: telegram/_bot.py (hunk 1 above)
Comment (Poolitzer) [reply telegram/_bot.py:9353]: The name

Comment (harshil21) [reply telegram/_bot.py:9352]: that's an optional argument tho. What happens when it's not passed? What is being edited?

File: docs/source/telegram.reactiontypepaid.rst [hunk 4]
```diff
@@ -0,0 +1,6 @@
+ReactionTypePaid
+================d
```
Comment (harshil21) [inline docs/source/telegram.reactiontypepaid.rst:2]:
```suggestion
================
```

File: telegram/_bot.py [hunk 5]
```diff
@@ -9274,8 +9282,119 @@ async def send_paid_media(
...
+        connect_timeout: ODVInput[float] = DEFAULT_NONE,
+        pool_timeout: ODVInput[float] = DEFAULT_NONE,
+        api_kwargs: Optional[JSONDict] = None,
+    ) -> ChatInviteLink:
+        """
+        Use this method to edit a subscription invite link created by the bot. The bot must have
+        :attr:`telegram.ChatPermissions.can_invite_users` administrator right.
+
+        .. versionadded:: NEXT.VERSION
+
```
Comment (harshil21) [inline telegram/_bot.py:9365]: we should probably add a Tip here saying to pass the name arg otherwise the method is useless

Comment (Bibo-Joshi) [reply telegram/_bot.py:9365]: I would expect that not passing the name will just remove the name. I can try to check

Comment (Bibo-Joshi) [reply telegram/_bot.py:9365]: Jup, that's what happens. Adding a tip about it.

**Review by harshil21 — APPROVED** [review]
lgtm

**Pull request merged**